pom.xml

```diff
@@ -72,6 +89,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.10.19</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```

Comment:
As I was prototyping the unit test/s, I found [mockito](http://mockito.org/) to have better support for what I was trying to do.  I'll be sure to use [wiremock](http://wiremock.org/) first if it supports what is needed to implement the test.